api/base/filters.py

```diff
@@ -25,16 +25,26 @@ def sort_multiple(fields):
     fields = list(fields)
     def sort_fn(a, b):
         while fields:
-            field = fields.pop(0)
-            a_field = getattr(a, field)
-            b_field = getattr(b, field)
-            if a_field > b_field:
-                return 1
-            elif a_field < b_field:
-                return -1
+            field = fields[0]
```

Comment:
This will cause an infinite loop if two `objects` with the same `field` are encountered. Please either change it back to `.pop`, or use a `for` loop (e.g. `for field in fields:`). Also, please add a test case that sorts two objects with equal fields.